core/bounds.gms

```diff
@@ -272,10 +272,35 @@ loop(te$(sameas(te,"spv") OR sameas(te,"wind") OR sameas(te,"windoff")),
 
 $ENDIF.WindOff
 
+***CG: fix CHA gas power capacities: 97 GW by September 2020 (Oxford Institute for Energy Studies, Natural gas in China’s power sector: Challenges and the road ahead, >60% are co-generation, rest are peaking)
+*** for 2018-2022, take 90GW, 90GW*0.6=54, the rest is split between ngcc and ngt
```

Comment:
please, don't use regioncodes in REMIND GAMS code. Make a new parameter for that or use an existing one and get the info into REMIND via input data. Alternatively, you can introduce a general rule which holds for all regions.

Reply:
Thanks, I merged this into mrremind now